api/proto/config/devmodel.proto

```diff
@@ -47,19 +47,19 @@ message sWAdapterParams {
 
 // systemAdapters, are the higher l2 concept built on physicalIOs.
 // systemAdapters, gives all the required bits to turn the physical IOs
-// into useful IP endpoints
+// into useful IP endpoints.
 // These endpoints can be further used to connect to controller or
 // can be shared between workload/services running on the node.
 message SystemAdapter {
         // name of the adapter; hardware-specific e.g., eth0
         string name = 1;
 
         // deprecated; need level 2 spec. sWAdapterParams allocDetails = 20;
-
         // this is part of the freelink group
-        // deprecate: have a separate device policy object in the API
+        // DEPRECATED by PhyIoAdapter.usagePolicy.
```

Comment:
Can you comment out the definition as well?

Reply:
Looks like no one is using this anyways.. Will remove this completely and just keep the number reserved in the comment.

Reply:
Better to see all of the changes. 
Sometimes I've deferred pushing the separate commit of the generated files to make life easier for the code reviewers, but there are no technical constraints to do all of it at once.

Reply:
Ok Cool.. I committed all the changes.

freeUplink -- I need to make a few changes.. will do it in a separate commit.. along with the support for the new policy fields..